paddle/fluid/operators/fake_quantize_op.cu

```diff
@@ -138,9 +138,9 @@ __global__ void ClipAndQuantDequantKernel(const T* in, const T* scale,
   int tid = threadIdx.x;
 
   T s = scale[0];
+  T inv_s = inverse(s);
   for (int i = bid; i < n; i += blockDim.x * gridDim.x) {
```

Comment:
Just to point out，the iterator variable i may become > INT_MAX (2G) if `blockDim.x * gridDim.x > 1G`.
You could keep it and we are going to fix that case uniformly in Paddle.

Reply:
Thanks for your suggestion.

Reply:
Qiuliang's comment is a great suggestion

Reply:
Yes, I will consider about it.